install.sh

```diff
@@ -5,7 +5,7 @@ pip install -r requirements.txt
 mkdir deps
 cd deps
 
-git clone -b dev_jeff https://github.com/BerkeleyAutomation/perception.git
+git clone https://github.com/BerkeleyAutomation/perception.git
```

Comment:
@visatish: this should probably be in a separate PR/branch, but I'm adding here anyway. `dev_jeff` is behind master, but I'm not sure of any other breaking changes this might introduce.